view_student.html

```diff
@@ -1,13 +1,32 @@
-<!--
-This file will provide the backbone for viewing student info.
--->
+<!DOCTYPE html>
 <html>
-  <head>
-  <title>View Student</title>
-  <link rel="stylesheet" type="text/css" href="css/upls_styles.css">
-  <script src="js/main_menu.js"></script>
-  </head>
-  <body onload="mainMenu()">
-    <div class="main_menu" id="main_menu"></div>
-  </body>
+<head>
+<title>View Student Information</title>
+<link href="styles.css" rel="stylesheet" type="text/css" media="all">
```

Comment:
This is not a file in the repo. If you have your own css for this page, add it to the pr, otherwise remove this reference.